### PR TITLE
Add docker setup instructions

### PR DESCRIPTION
**Pull request by NullSoldier**

### CODE REVIEW

File: docs/onboarding/installation.md [hunk 1]
```diff
@@ -31,10 +31,19 @@ Iron Fish is now ready to use. Follow the [next step](new_node.md) of the tutori
 Windows installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source)
```
Comment (dguenther) [inline docs/onboarding/installation.md:31]: Docker link here too maybe?
```suggestion
Windows installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source) or [use Docker](#using-docker).
```

File: docs/onboarding/installation.md [hunk 2]
```diff
@@ -31,10 +31,19 @@ Iron Fish is now ready to use. Follow the [next step](new_node.md) of the tutori
...
 
 ## Using Docker
-Docker images will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source)
+
+Pull down the docker image from the github registry and run the node and mount the node's data directory into your home directory
+```sh
+docker run --rm --tty --interactive --network host --volume $HOME/.ironfish:/root/.ironfish ghcr.io/iron-fish/ironfish:latest
+```
+
+Now check the status of your node using ICP over the docker host network, you can also use the CLI / RPC layer over TCP as well by starting your node with `--rpc.tcp` which connects over the default port port 8020.
```
Comment (dguenther) [inline docs/onboarding/installation.md:43]:
```suggestion
Now check the status of your node using IPC. You can also use the RPC layer over TCP by starting your node with `--rpc.tcp`, which connects over the default port 8020.
```

File: docs/onboarding/installation.md [hunk 3]
```diff
@@ -31,10 +31,19 @@ Iron Fish is now ready to use. Follow the [next step](new_node.md) of the tutori
 Windows installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source)
 
 ## Linux
-Linux installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source)
+Linux installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source) or [use Docker](#using-docker)
 
 ## Using Docker
-Docker images will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source)
+
+Pull down the docker image from the github registry and run the node and mount the node's data directory into your home directory
```
Comment (dguenther) [inline docs/onboarding/installation.md:38]:
```suggestion
Run the Docker image from the Github registry, mounting the node's data directory into your home directory and using the host network.
```

File: docs/onboarding/installation.md [hunk 4]
```diff
@@ -31,10 +31,19 @@ Iron Fish is now ready to use. Follow the [next step](new_node.md) of the tutori
 Windows installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source)
 
 ## Linux
-Linux installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source)
+Linux installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source) or [use Docker](#using-docker)
```
Comment (dguenther) [inline docs/onboarding/installation.md:34]:
```suggestion
Linux installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source) or [use Docker](#using-docker).
```

**Review by dguenther — APPROVED** [review]

File: docs/onboarding/installation.md (hunk 4 above)
Comment (NullSoldier) [reply docs/onboarding/installation.md:34]: done

File: docs/onboarding/installation.md (hunk 3 above)
Comment (NullSoldier) [reply docs/onboarding/installation.md]: done

File: docs/onboarding/installation.md (hunk 2 above)
Comment (NullSoldier) [reply docs/onboarding/installation.md]: done

**Pull request merged**